### PR TITLE
botan 3.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  build/include/botan/ec_point.h:36:13: error: expected unqualified-id
        using enum EC_Point_Format;
              ^
```

**Comment by chenrui333** [comment]
logged an issue on qca side for botan v3 support, https://invent.kde.org/libraries/qca/-/issues/13

File: Formula/botan.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ class Botan < Formula
   depends_on "pkg-config" => :build
   depends_on "python@3.11"
   depends_on "sqlite"
+  depends_on xcode: "13.3"
```
Comment (carlocab) [inline Formula/botan.rb:28]: Not sure this is right, since this will require a full Xcode install, when it seems like a new enough CLT suffices.

Comment (chenrui333) [reply Formula/botan.rb:28]: upstream needs xcode 13.3, so I figured this dependency declaration would be the route, any thoughts?

Comment (SMillerDev) [reply Formula/botan.rb:28]: Does it not work with the CLT?

Comment (carlocab) [reply Formula/botan.rb]: It should. Let's just use `fails_with` instead of depending on a full Xcode installation.

**Review by p-linnane — DISMISSED** [review]

**Pull request merged**